### PR TITLE
Revert .NET site to be at /csharp

### PR DESCRIPTION
**Pull request by MariusVolkhart**
We have existing links point to /csharp that we've sent to customers. This seems easier than setting up page redirects.

### CODE REVIEW

File: .github/workflows/deploy_docs.yml [hunk 1]
```diff
@@ -44,7 +44,7 @@ jobs:
         run: python3 -m pip install -r requirements-sphinx.txt
 
       - name: Build .NET site
-        run: sphinx-build -b html -t csharp dotnet/ root/dotnet
+        run: sphinx-build -b html -t csharp dotnet/ root/csharp
```
Comment (MariusVolkhart) [inline .github/workflows/deploy_docs.yml:47]: I'm going to do another PR in a bit to combine these two files

**Review by isaac57 — APPROVED** [review]
